### ISSUE TITLE
[Question] 请问BERTEmbedding和Keras模型的正确结合姿势

### ISSUE THREAD

**Issue by xyk1994** [BrikerMan/Kashgari#286]
**You must follow the issue template and provide as much information as possible. otherwise, this issue will be closed.
请按照 issue 模板要求填写信息。如果没有按照 issue 模板填写，将会忽略并关闭这个 issue**

## Check List
Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

**You can post pictures, but if specific text or code is required to reproduce the issue, please provide the text in a plain text format for easy copy/paste.**

- [x] I have searched in [existing issues](https://github.com/BrikerMan/Kashgari/issues?utf8=%E2%9C%93&q=is%3Aissue+) but did not find the same one.
- [x] I have read the [documents](https://kashgari.bmio.net)

## Environment

- OS [e.g. Mac OS, Linux]: Linux
- Python Version: 3.6
- requirements.txt:

```txt
[Paste requirements.txt file here]
```

## Question

您好，问几个小白问题。。。网上资料确实少，周围也没人能问，只能直接问专家本人了。。。

1. BERTEmbedding能直接用在keras.model 里吗？
之前尝试过Glove，可以直接用keras的Embedding层。但是我直接把BERTEmbedding层替换之后发现BERTEmbedding.embed的输入输出是np.ndarray，model里input之后的数据就是tensor了，好像不能直接做 BERTEmbedding？这个时候报错是‘Tensor objects are only iterable when eager execution is enabled. To iterate over this tensor use tf.map_fn.’

2. 之后我按照教程的自定义模型结构改写了一下之后，模型大概如下
def get_model():
    print('Build model ...')

    W_bert_embed_essay = BERTEmbedding('uncased_L-12_H-768_A-12',
                                       trainable=True,
                                       sequence_length=max_len_bert,
                                       task=kashgari.CLASSIFICATION
                                       )

    embed_model = W_bert_embed_essay.embed_model

    lstm1 = LSTM(64, dropout=0.5, recurrent_dropout=0.5, return_sequences=True)(embed_model.output)
    lstm2 = LSTM(64, return_sequences=False)(lstm1)
    output = Dense(1, activation='sigmoid')(lstm2)
    model = Model(inputs=embed_model.inputs, outputs=output)

    model.summary()

    return model

这个时候报错的'Node' object has no attribute 'output_masks'
但是如果我 from kashgari.layers import L 
然后把LSTM和Dense替换成L.LSTM，L.Dense之后，报错 object of type 'LSTM' has no len()
我寻思这L不是keras.layers吗，为什么我直接从keras.layers调用的LSTM和L.LSTM报错为什么不一样？
然后这个问题该怎么解决啊。。。求指导。

之前做特征提取的时候，是把词向量提取出来后，将词向量当做输入输入到keras.model里面。
现在想尝试下微调，不知道具体该怎么做。

**Comment by BrikerMan** [comment]
其实目前的 Kashgari 是基于 tf.keras，所以只能和 tf.keras 模型结合，不能用 keras 的层。
